LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2021 Contrast Security, Inc.
+Copyright 2014 - 2022 Contrast Security, Inc.
```

Comment:
I don't think we're supposed to have this range in here. I'll send you the internal guidance I'm working off of.

Reply:
addressed